modules/openapi-generator/src/main/resources/rust-server/Cargo.mustache

```diff
@@ -35,38 +35,38 @@ homepage = "{{.}}
 default = ["client", "server"]
 client = [
 {{#apiUsesMultipart}}
-    "mime_0_2",
```

Comment:
This whitespace isn't consistent with the other rust templates. Happy to revert this bit though.